test/reactionsystem_core/reactionsystem.jl

```diff
@@ -126,15 +127,11 @@ let
           defs
 
     u0map = [A => 5.0]
-    pmap = [k[1] => 5.0]
+    kvals[1] = 5.0
+    pmap = [k => kvals]
     prob = ODEProblem(rs, u0map, (0, 10.0), pmap)
     @test prob.ps[k[1]] == 5.0
     @test prob.u0[1] == 5.0
-    u0 = [10.0, 11.0, 12.0, 13.0]
-    ps = [float(x) for x in 100:119]
-    prob = ODEProblem(rs, u0, (0, 10.0), ps)
-    @test  [prob.ps[k[i]] for i in 1:20] == ps
-    @test prob.u0 == u0
```

Comment:
These are not appropriate inputs since they aren't mappings, hence I removed them.